cmd/calyptia/update_pipeline_cluster_objects.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	cloud "github.com/calyptia/api/types"
+)
+
+func newCmdUpdatePipelineClusterObjects(config *config) *cobra.Command {
+	var pipelineKey string
+	var clusterObjectKey string
+	var environment string
+	var encrypt bool
+
+	cmd := &cobra.Command{
+		Use:   "pipeline_cluster_objects",
+		Short: "Update a file from a pipeline by its name",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var environmentID string
+			if environment != "" {
+				var err error
+				environmentID, err = config.loadEnvironmentID(environment)
+				if err != nil {
+					return err
+				}
+			}
+
+			pipelineID, err := config.loadPipelineID(pipelineKey)
+			if err != nil {
+				return err
+			}
+
+			clusterObjectID, err := config.loadClusterObjectID(clusterObjectKey, environmentID)
+			if err != nil {
+				return err
+			}
+
+			err = config.cloud.UpdatePipelineClusterObjects(config.ctx, pipelineID, cloud.UpdatePipelineClusterObjects{
```

Comment:
Really, this doesn't update a cluster object, but de/attaches objects to a pipeline... 🤔

Reply:
It indeed does.